src/js/components/Grommet/Grommet.js

```diff
@@ -1,15 +1,15 @@
-import React, { Component } from 'react';
+import React, { forwardRef, useEffect, useMemo, useState } from 'react';
 import { createGlobalStyle } from 'styled-components';
 
 import {
+  ContainerTargetContext,
   ResponsiveContext,
   ThemeContext,
-  ContainerTargetContext,
 } from '../../contexts';
 
 import {
-  backgroundIsDark,
   deepMerge,
+  backgroundIsDark,
```

Comment:
why did this prop move a line?

Reply:
the import just moved since I pretty much re-worked the entire file

Reply:
I think we should lean towards alphabetized order.